### ISSUE TITLE
AttentionAgent: Is there GPU support? How long approximately is training?

### ISSUE THREAD

**Issue by zeyuyun1** [google/brain-tokyo-workshop#33]
Hi, thank you so much for the awesome works!

It seems like there is no GPU support for AttentionAgent. Or am I missing something? How long would it take locally for CarRacing to reach 900 reward like mentioned in the paper? @lerrytang

**Comment by lerrytang** [comment]
Hi, thanks for being interested!

I've created another [branch](https://github.com/google/brain-tokyo-workshop/tree/simplified_aa/AttentionAgent) that contains some simplified code for training on a local machine.
On a 96 CPU machine, I was able to train the model for 10 hours and get 854 averaged over 100 rollouts (see `pretrained/carracing/test_solution.txt`), I guess that if I had let it run longer it can hit 900.

You can also train an agent on GPU, please see the README for command.
I wanted to test and respond to you quickly, so I've only tested GPU training on the MLPSolution.
Please see the MLPSolution example in `torch_solutions.py` and make similar changes to AttentionAgent.
If you have any problems, please let me know.
